src/FOSRestExtraBundle/EventListener/ParamFetcherListener.php

```diff
@@ -75,16 +53,14 @@ public function setErrorCode($code)
 
     /**
      * Core controller handler.
-     *
-     * @throws HttpException
      */
     public function onKernelController(ControllerEvent $event): void
     {
-        if ($event->isMasterRequest()) {
+        if ($event->isMainRequest()) {
```

Comment:
I believe this make the bundle dependent on symfony/http-kernel>=5.3 and it could be explicit in composer.json, since friendsofsymfony/rest-bundle:3.3 still support version 4.4

Reply:
Done. Thanks